resources/metadata/abapAddonAssemblyKitCheckCVs.yaml

```
@@ -0,0 +1,68 @@
+metadata:
+    name: abapAddonAssemblyKitCheckCVs
+    description: This step checks the validity of Software Component Versions.
+    longDescription: |
+      This steps takes a list of Software Component Versions from the addonDescriptorFileName and checks whether they exist or are a valid successor of an existing Software Component Version.
+      It resolves the dotted version string into version, support package level and patch level and writes it to the commonPipelineEnvironment.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapAddonAssemblyKitCredentialsId
+        description: Credential stored in Jenkins for Addon Assembly Kit as a Service System (AAKaaS)
+        type: jenkins
+    params:
+      - name: abapAddonAssemblyKitEndpoint
+        type: string
+        description: Base URL to the Addon Assembly Kit as a Service System (AAKaaS)
```

I would express it like this:

```suggestion
        description: Base URL to the Addon Assembly Kit as a Service (AAKaaS) system
```

---

Yes looks better :-) will be part of my next commit